specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2022-11-30-preview/definitions/MigrateMySqlCommon.json

```diff
@@ -0,0 +1,95 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Database Migration Service Resource Provider",
+    "version": "2022-11-30-preview"
+  },
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {},
+  "definitions": {
+    "MySqlBinlogPositionInput": {
+      "type": "object",
+      "description": "MySQL binlog reference",
+      "properties": {
+        "filename": {
+          "type": "string",
+          "description": "Specifies the MySQL binlog file name"
+        },
+        "position": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Specifies the position in the binlog file"
+        }
+      }
+    },
+    "MySqlBinlogPositionOutput": {
+      "type": "object",
+      "description": "MySQL binlog reference",
+      "properties": {
+        "name": {
```

Comment:
>name [](http://example.com/codeflow?start=9&length=4)

If this property is anywhere in the property bag of an ARM resource type, it's not allowed to be named "name". The top-level property names (e.g. id, name, type, identity, etc.) are not allowed in the property bag.

Reply:
What does it mean for it to be in the property bag?